backend/src/packages/projects/project.repository.ts

```diff
@@ -17,11 +19,18 @@ class ProjectRepository
     return projects.map((project) => ProjectEntity.initialize(project));
   }
 
-  public async findByUserId(id: number): Promise<ProjectEntity[]> {
+  public async findByUserId(
+    { query }: ProjectSearchParameters,
+    id: number,
+  ): Promise<ProjectEntity[]> {
     const projects = await this.projectModel
       .query()
       .where('userId', id)
-      .execute();
+      .andWhere((builder) => {
+        if (query) {
+          void builder.whereRaw('name ilike ?', [`%${query}%`]);
```

Comment:
Can't it be like this? - this way knex will escape the value by itself without passing params with `?`
```suggestion
          void builder.where('name', 'ilike', `%${query}%`);
```